l1-contracts/deploy-scripts/upgrade/v30/EcosystemUpgrade_v30.s.sol

```diff
@@ -117,14 +107,28 @@ contract EcosystemUpgrade_v30 is Script, DefaultEcosystemUpgrade {
         return super.getCreationCalldata(contractName, isZKBytecode);
     }
 
-    function deployUsedUpgradeContract() internal override returns (address) {
+    function deployUsedUpgradeContract() internal returns (address) {
         return deploySimpleContract("L1V29Upgrade", false);
     }
 
-    function getInitializeCalldata(string memory contractName) internal virtual override returns (bytes memory) {
-        if (compareStrings(contractName, "MessageRoot")) {
-            return abi.encodeCall(MessageRoot.initializeL1V30Upgrade, ());
-        }
-        return super.getInitializeCalldata(contractName);
-    }
+    // CTM Upgrade Functions
```

Comment:
why comment it out?

Reply:
We have to split the current upgrade into EcosystemUpgrade, CTM Upgrade, Gateway upgrade. 
Currently this functions are from CTM upgrade. not from EcoSystem